WordPress/src/main/java/org/wordpress/android/ui/CollapseFullScreenDialogFragment.java

```diff
@@ -257,15 +263,16 @@ private void initBuilderArguments() {
      */
     private void initToolbar(View view) {
         Toolbar toolbar = view.findViewById(R.id.full_screen_dialog_fragment_toolbar);
+
+        ElevationOverlayProvider elevationOverlayProvider = new ElevationOverlayProvider(view.getContext());
+        float appbarElevation = getResources().getDimension(R.dimen.appbar_elevation);
+        int elevatedColor = elevationOverlayProvider.compositeOverlayWithThemeSurfaceColorIfNeeded(appbarElevation);
+        toolbar.setBackgroundColor(elevatedColor);
+
         toolbar.setTitle(mTitle);
         toolbar.setNavigationContentDescription(R.string.description_collapse);
         toolbar.setNavigationIcon(ContextCompat.getDrawable(view.getContext(), R.drawable.ic_chevron_down_white_24dp));
-        toolbar.setNavigationOnClickListener(new View.OnClickListener() {
-            @Override
-            public void onClick(View view) {
-                onCollapseClicked();
-            }
-        });
+        toolbar.setNavigationOnClickListener(view1 -> onCollapseClicked());
```

Comment:
I would consider to not use the numbering in the variable name (something like a simple `v` maybe?). Same in `CommentDetailFragment`. (I think the `view1` naming is the one created by AS to avoid the clash with the function parameter but I would consider to modify it as well, let me know wdyt 😊)

Reply:
Good catch! Yes, AS creates those sometimes, even though it's inside a function. Weird.